lib/llm/src/hub.rs

```diff
@@ -36,68 +32,120 @@ fn is_weight_file(filename: &str) -> bool {
         || filename.ends_with(".ckpt.index")
 }
 
-/// Attempt to download a model from Hugging Face
+/// Attempt to download a model from Hugging Face using ModelExpress client with hf-hub fallback
 /// Returns the directory it is in
 /// If ignore_weights is true, model weight files will be skipped
 pub async fn from_hf(name: impl AsRef<Path>, ignore_weights: bool) -> anyhow::Result<PathBuf> {
     let name = name.as_ref();
+    let model_name = name.display().to_string();
+    
+    // Get ModelExpress server endpoint from environment or use default
+    let endpoint = env::var(MODEL_EXPRESS_ENDPOINT_ENV_VAR)
+        .unwrap_or_else(|_| DEFAULT_MODEL_EXPRESS_ENDPOINT.to_string());
+    
+    // Create client configuration
+    let config = ClientConfig::for_testing(&endpoint);
```

Comment:
`for_testing` doesn't sound right? (I know this is still draft).

Reply:
you are right, needed default constructor